src/frontend/components/Banners/DinosaursBanner/DinosaursBanner.css

```diff
@@ -0,0 +1,173 @@
+/* Dinosaurs Banner */
+
+.home-screen-row {
+  height: 100vh;
+}
+
+.text-banner {
+  width: 100vw;
+  opacity: 0.8;
+}
+
+.dinosize-chart {
+  background: #55a4b1;
+  background: #101212;
+  background: #ffffff;
+  background: #00919c;
+  background: #ced02d;
+  background: #c6a20b;
```

Comment:
I am not sure what you are trying to achieve here 
last background will overwrite the previous background
anyway using variables will be good practice

Reply:
Ah yes! Totally missed this.
I  was trying out some bg colors & for got remove others.